integration/docker/package_manager_test.go

```diff
@@ -30,6 +30,20 @@ func tryPackageManagerCommand(container string, command []string, expectedExitCo
 	return exitCode
 }
 
+func distroID() string {
```

Comment:
there is already a similar function in `tests/functional/state_test.go`. How about move it to `tests/distro.go` and import it here and in `tests/functional/state_test.go` ?